assets/constants.ts

```diff
@@ -6490,7 +6490,7 @@ export default {
   // fields must be at most 1500 bytes, and UTF-8 encoded characters can be
   // up to 4 bytes long.
   "MAX_COMMIT_MESSAGE_LENGTH": 375,
-  "MAX_REVIEW_MESSAGE_LENGTH": 10000,
+  "MAX_REVIEW_MESSAGE_LENGTH": 2000,
```

Comment:
Hi @Ari1009, could you check if this contant is used in other places besides the "feedback" input in the original issue? We may still want the original message length in other places. If so, we will need to use a separate constant with 2000 limit for feedback specifically. Thanks!

Reply:
@Ari1009 have you checked all these files?

![image](https://github.com/oppia/oppia/assets/135348086/82f8484c-c058-4c7e-b1b8-f28703fae469)

Reply:
Hi @Ari1009, as you can see, the suggestion review component uses the same value. This is used in the contributor dashboard page for a different purpose — it defines the max length of review message in the suggestion reviewer interface. I’m not sure if we should be updating the value there as well. Perhaps @chris7716 from the CD team can confirm.

Reply:
I had a chat with Sean, it might be OK to keep the updated value for the review message as well. Thanks!